src/components/ComparisonMode.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import { ReactComponent as BackIcon} from "../assets/search-icons/back.svg";
+import "../css/collapsible.css";
+import "../css/sidebar.css";
+import { Link } from "react-router-dom";
+import Collapsible from "react-collapsible";
+import LocationCompare from "./LocationToCompare";
+
+class ComparisonMode extends React.Component {
```

Comment:
This component could use a more specific name than ComparisonMode. ComparisonMode could apply to any of the components we'll make for that mode. It's the comparison mode version of sidebar content, so potentially ComparisonSidebarContent.